argo/src/Argo.hs

```diff
@@ -339,6 +361,83 @@ instance JSON.ToJSON Request where
       "id"      .= view requestID     req <>
       "params"  .= view requestParams req
 
+data Response a
+  = Response { _responseID :: RequestID
+             , _responseAnswer :: a
+             , _responseStdOut :: String
+             , _responseStdErr :: String
+             } deriving (Eq, Ord, Show)
+
+responseID :: Simple Lens (Response a) RequestID
```

Comment:
`Simple Lens` isn't compatible with new GHCs. Can this be `Lens'` instead, which has the same meaning?

Reply:
I would expect so! I'll change it.

Reply:
Thanks!